src/components/Modal/tests/Modal.test.tsx

```diff
@@ -186,33 +186,6 @@ describe('<Modal>', () => {
 
       expect(modal.find(Badge).exists()).toBe(false);
     });
-
-    it('triggers an onTransitionEnd prop', () => {
-      const modal = mountWithAppProvider(
-        <Modal
-          onClose={jest.fn()}
-          open
-          secondaryActions={[{content: 'Discard', onAction: jest.fn()}]}
-          onTransitionEnd={jest.fn()}
-        />,
-      );
-
-      trigger(modal, 'onTransitionEnd');
-      expect(modal.prop('onTransitionEnd')).toHaveBeenCalledTimes(1);
-    });
-
-    it('triggers onTransitionEnd from Dialog', () => {
-      const modal = mountWithAppProvider(
-        <Modal
-          open
-          onClose={jest.fn()}
-          secondaryActions={[{content: 'Discard', onAction: jest.fn()}]}
-          onTransitionEnd={jest.fn()}
-        />,
-      );
-      trigger(modal.find(Modal.Dialog), 'onEntered');
-      expect(modal.prop('onTransitionEnd')).toHaveBeenCalledTimes(1);
-    });
```

Comment:
Why are these getting deleted?

Reply:
They stopped working since they use `prop()` on the wrapper component. And I didn't fix them since they don't provide much value and I can't think of an appropriate test to use here without slightly refactoring the component.

```
  private handleEntered = () => {
    const {onTransitionEnd} = this.props;
    if (onTransitionEnd) {
      onTransitionEnd();
    }
  };
```

Should be removed and `onTransitionEnd` should be passed to `Dialog`. I'll make a note to fix this in a follow up pr